src/core/Form/TextInput/TextInput.md

```diff
@@ -2,6 +2,7 @@
 import { TextInput } from 'suomifi-ui-components';
 <>
   <TextInput
+    className="testClassName"
```

Comment:
does this serve a purpose or is it a relic from manual testing?

Reply:
It was there to demonstrate custom styling, but decided to remove it since it has no effect here.